packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.routing.js

```diff
@@ -0,0 +1,12 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('overTheBoxes.overTheBox.details', {
+    url: '/details',
+    views: {
+      otbView: 'overTheBoxDetails',
+    },
+    resolve: {
+      serviceName: /* @ngInject */ ($transition$) =>
```

Comment:
Maybe it makes more sense to declare it in the parent state as it is where the param is declared and the siblings can benefit from this injection